### PR TITLE
[main] Add Konflux configurations

### PR DESCRIPTION
**Pull request by serverless-qe**
Add Konflux components and pipelines

### CODE REVIEW

**Comment by pierDipi** [comment]
prowgen is deleting 1.35 openshift/release config https://github.com/openshift/release/pull/59243

File: .konflux/releaseplanadmissions/serverless-operator-136-1350-fbc-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    releaseNotes:
+      product_id: 579
+      product_name: "OpenShift Serverless"
+      product_version: "1.35.0"
+      references:
+        - https://docs.redhat.com/en/documentation/red_hat_openshift_serverless
+      solution: "The container images provided by this update can be downloaded from the Red Hat container registry at registry.redhat.io"
+      description: "Release 1.35.0 of OpenShift Serverless"
+      topic: "Release 1.35.0 of OpenShift Serverless"
+      synopsis: "OpenShift Serverless Release 1.35.0"
```
Comment (creydr) [inline .konflux/releaseplanadmissions/serverless-operator-136-1350-fbc-prod.yaml:29]: this should be 1.36

File: .konflux/releaseplans/serverless-operator-136-1350-prod.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    release.appstudio.openshift.io/standing-attribution: 'true'
+  name: serverless-operator-136-1350-prod
+spec:
+  releaseNotes:
+    type: "Product Enhancement Advisory"
+    synopsis: "Red Hat OpenShift Serverless Release"
+    topic: |
+      The 1.35.0 GA release of Red Hat OpenShift Serverless Operator.
+      For more details see [product documentation](https://docs.redhat.com/documentation/red_hat_openshift_serverless).
+    description: "The 1.35.0 release of Red Hat OpenShift Serverless Operator."
```
Comment (creydr) [inline .konflux/releaseplans/serverless-operator-136-1350-prod.yaml:16]: should also be 1.36

File: .konflux/releaseplanadmissions/serverless-operator-136-1350-fbc-prod.yaml (hunk 1 above)
Comment (pierDipi) [reply .konflux/releaseplanadmissions/serverless-operator-136-1350-fbc-prod.yaml:29]: Yeah, it will change eventually

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**